scripts/setup_release_mirror.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "generating ssl certs"
+./usr/local/app/scripts/generate_ssl.sh
+echo "custom /etc/hosts"
+echo '127.0.0.1 github.com' >> /etc/hosts
+echo "running nginx"
+nginx
```

Comment:
Does this run nginx in the background ? If not I'm confused how the script continue running istio install.

Reply:
Yes, by default nginx runs in the background, see https://nginx.org/en/docs/ngx_core_module.html#daemon